docs/docs/basic-hardware-software-requirements.md

```diff
@@ -2,6 +2,34 @@
 title: Basic Hardware and Software Requirements
 ---
 
-This is a stub. Help our community expand it.
+Find the latest hardware and software requirements for building with Gatsby. Note: this doc will evolve as the framework evolves, so if you find something that's outdated please [open an issue with your findings](/contributing/how-to-file-an-issue/).
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your pull request gets accepted.
+## Operating Systems
+
+- Mac OSX Sierra or later (10.12)
+- Windows 10 ([setup instructions](/docs/gatsby-on-windows/))
+- Linux ([multiple distributions](/docs/gatsby-on-linux/))
+
+## Memory
+
+This will be variable depending on the size of your site. Gatsby sites have been known to work from 500mb to 1GB of RAM.
+
+## Node.js ecosystem
+
+To develop with Gatsby, you'll need to install:
+
+- Node 10.13.0 (LTS) or higher
+  - See [Gatsby notice of dropping Node 8](/blog/2020-03-20-dropping-support-for-node-8)
+- [npm](https://www.npmjs.com/) or [Yarn](https://yarnpkg.com/) package manager to install the [Gatsby CLI](/docs/gatsby-cli/) and site dependencies.
+  - npm is recommended for most developers.
+  - Yarn is used for authoring Gatsby themes.
```

Comment:
I would recommend Yarn 1. You could say that while Yarn 2 should work with our core plugins, some third party plugins do not.